build-system/tasks/presubmit-checks.js

```diff
@@ -397,6 +405,7 @@ const forbiddenTerms = {
   'isTrustedViewer': {
     message: requiresReviewPrivacy,
     whitelist: [
+      'extensions/amp-bind/0.1/bind-impl.js',
```

Comment:
Seems like this is an unrelated change. Revert.

Reply:
Found this! https://github.com/ampproject/amphtml/pull/20822/files